styles/components/account/account-user-name.module.scss

```diff
@@ -0,0 +1,93 @@
+.account-user-name {
+	margin-bottom: 20px;
+	position: relative;
+
+	input[type="text"] {
+		overflow: auto;
+		padding-bottom: 3px;
+		font-family: Kanit;
+		width: 275px;
+		height: 55px;
+		position: relative;
+		background-color: transparent;
+		text-align: center;
+		font-size: 3.5rem;
+		font-weight: 500;
+		color: var(--secondary-color);
+	}
+
+	.user-name-input-save {
+		border: none;
+		border-bottom: 4px var(--secondary-color) solid;
+	}
+
+	.user-name-input-edit {
+		border: none;
+	}
+
+	.user-name-view {
+		line-height: 3.5rem;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		text-wrap: nowrap;
+		display: inline-block;
+		padding-bottom: 3px;
+		font-family: Kanit;
+		width: 275px;
+		position: relative;
+		text-align: center;
+		font-size: 3.5rem;
+		font-weight: 500;
+		color: var(--secondary-color);
+	}
+
+	button {
+		font-family: Kanit;
+		position: absolute;
+		right: -150px;
+		cursor: pointer;
+		width: 125px;
+		height: 50px;
+		text-align: center;
+		box-sizing: border-box;
+		border-radius: 8px;
+		border: solid 3px var(--secondary-color);
+		background-color: var(--secondary-color);
+		color: var(--primary-color);
+		font-size: 1.5rem;
+		font-weight: 500;
+	}
+}
+
+@media only screen and (max-width: 600px) {
+	.account-user-name {
+		margin-bottom: 20px;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		gap: 25px;
+
+		input[type="text"] {
+			width: 180px;
+			height: 45px;
+			font-size: 2.5rem;
+			padding-bottom: 0px;
+		}
+
+		.user-name-view {
```

Comment:
repeating declaration with the input put in a chaining selector

Reply:
noted